### ISSUE TITLE
1.0.11rc1 release vagrant testing results

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#2751]
1.0.11rc1 release vagrant testing results:
```
darwin64 ok

freebsd64 ok
netbsd64 "ok" (timing diff link1)
openbsd64 n/a - shell invalid?

wheezy64 ok
wheezy32 ok
jessie64 ok

centos7_64 "ok" EISDIR with fakeroot. acl test uebel user missing. 
centos6_32 "ok" EISDIR with fakeroot. acl test uebel user missing.
centos6_64 "ok" EISDIR with fakeroot. acl test uebel user missing.

trusty64 MemoryError, XDISTN=1 -> ok
xenial64 ok
```

**Comment by ThomasWaldmann** [comment]
TODO:
- check vagrant memory vs. XDISTN (see also master)
- fix openbsd shell

**Comment by ThomasWaldmann** [comment]
fixed by #2752
